### PR TITLE
docs: fix typo in navigation.json

### PR DESCRIPTION
**Pull request by aikithoughts**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
navigation.json says "toolTOP", not "toolTIP"

Issue Number: N/A

## What is the new behavior?
navigation.json says tooltip

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]

File: aio/content/navigation.json [hunk 1]
```diff
@@ -748,7 +748,7 @@
     },
     {
       "title": "Tutorials",
-      "tooltop": "End-to-end tutorials for learning Angular concepts and patterns.",
+      "tooltip": "End-to-end tutorials for learning Angular concepts and patterns.",
```
Comment (IgorMinar) [inline aio/content/navigation.json:751]: I liked "tooltop" better. 😜

**Review by IgorMinar — APPROVED** [review]
LGTM - except for the failing lint - can you please fix that! Thank you!

Reviewed-for: docs-marketing

**Comment by IgorMinar** [comment]
https://app.circleci.com/pipelines/github/angular/angular/14684/workflows/80ded117-2355-4557-a435-14c013c4f556/jobs/702325/parallel-runs/0/steps/0-108

**Review by IgorMinar — APPROVED** [review]
I love the new commit message description. Thanks for patiently explaining the context behind this change.

side-note: For quite some time I've been eyeing Colemak instead of Dvorak (which I learned at some point but not well enough to fully switch to it for day to day use). Why Colemak, because my dear friend @vojtajina has done the research and put in front of me a very convincing argument: https://medium.com/@vojtajina/my-keyboard-layout-is-better-than-yours-f23b19d1b046

File: aio/content/navigation.json (hunk 1 above)
Comment (IgorMinar) [reply aio/content/navigation.json:751]: Have you considered "toolbottom" as an alternative?

**Pull request closed without merging**